src/clients/apiClient.ts

```diff
@@ -49,6 +49,15 @@ export interface User {
   userType: UserType
 }
 
+export interface Like {
+  id: number,
+  date: Date,
+  whaleSightingId: number,
+  whaleSighting: WhaleSighting,
+  userId: number,
+  user: User
+}
```

Comment:
I think we will want to change this interface to have the same properties as the `LikeResponse` in the backend